docs/contributing/contributing_pl.md

```diff
@@ -33,7 +33,7 @@ Bruno jest rozwijane jako aplikacja desktopowa. Musisz załadować aplikację, u
 
 ### Lokalny Rozwój
 
-```bash
+````bash
```

Comment:
There's a typo - missed `` ``` `` for this `` ```bash ``. Because of that, your autoformat (or maybe git hooks) converted:

````
```bash
```shell
```
````

to

`````
````bash
```shell
````
`````

Reply:
Thanks for pointing out @Nikolai2038 

@jaideepghosh / @Nikolai2038 Can you raise a PR fixing the typo?

Reply:
I'll leave it to @jaideepghosh. I just stumbled on it too, when doing [this pull request](https://github.com/usebruno/bruno/pull/1037) (not closed yet). In the end, this was not related to my changes, so I did not include this fix.

Reply:
> I'll leave it to @jaideepghosh. I just stumbled on it too, when doing [this pull request](https://github.com/usebruno/bruno/pull/1037) (not closed yet). In the end, this was not related to my changes, so I did not include this fix.

Thanks @Nikolai2038 for highlighting this issue. @helloanoop I'll raise another PR to fix this.

Reply:
Awesome. Thanks @jaideepghosh !

Reply:
@helloanoop Please review this PR: https://github.com/usebruno/bruno/pull/2354
It has the fix for this auto formatting added by vs-code.